metabase/metabase
# Mantine Title

Closes https://github.com/metabase/metabase/issues/32441

Adds `Title` component with style overrides.

How to verify:
 - `yarn storybook`
 - Open http://localhost:6006/?path=/story/typography-title--default-story
 - You can change options in the default story, or go to Docs and explore pre-built examples
 - Please note that `truncute` won't work for the `Default` story because the enclosing block larger than the text

<img width="1071" alt="Screenshot 2023-08-17 at 18 35 23" src="https://github.com/metabase/metabase/assets/8542534/12fd6b4e-723a-4c98-a030-e8730decf52d">
